File: server/Models/Account.cs
using System.Text.Json.Serialization;

namespace allspice.Models;

public class Account
{
  [JsonPropertyName("sub")]
  public string Id { get; set; }
  [JsonPropertyName("name")]
  public string Name { get; set; }
  [JsonPropertyName("email")]
  public string Email { get; set; }
  [JsonPropertyName("picture")]
  public string Picture { get; set; }
}

File: client/src/models/Account.js
export class Account {
  /**
   * @typedef AccountData
   * @property {string} sub
   * @property {string} email
   * @property {string} name
   * @property {string} picture
   * 
   * @param {AccountData} data
   */
  constructor(data) {
    this.sub = data.sub
    this.email = data.email
    this.name = data.name
    this.picture = data.picture
    // TODO add additional properties if needed
  }
}
